.dockerignore

```diff
@@ -0,0 +1,8 @@
+.env
+.git
```

Comment:
I wonder if this would impact a future `rumi` integration?

Reply:
I see what you mean, if the `.git` directory is omitted, rumi will not have access to the history. However, I very much doubt that rumi will be run in the context of these docker containers which are intended as a production build for the deployment environment. Rumi will likely run in the context of a local directory (e.g. not in the container) or in a GitHub action, which will clone the entire repo including the .git history.